src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -236,6 +237,17 @@ def acs_install_cli(resource_group, name, install_location=None, client_version=
     else:
         raise CLIError('Unsupported orchestrator type {} for install-cli'.format(orchestrator_type))
 
+def _ssl_context():
+    if sys.version_info < (2, 8):
```

Comment:
`ssl. create_default_context` is new in `3.4`. We should cap the version at `3.4`. It is uncertain if the API will be available in `2.8`.